src/client/javascripts/App.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Sample React Native App
+ * https://github.com/facebook/react-native
+ * @flow
+ */
+
+import React, { Component } from 'react';
+import {
+    AppRegistry,
+    StyleSheet,
+    Text,
+    View,
+    Platform
+} from 'react-native';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+        titleText: 'Yoda',
+        os: 'OS',
```

Comment:
Is this meant to be the default string? Do we need to set this to 'OS', or can we just set it to an empty string like ''? I'd rather see an empty string than having some meaningless random constant like 'OS'.